code/game/objects/items/stacks/nanopaste.dm

```diff
@@ -16,6 +16,9 @@
 /obj/item/stack/nanopaste/attack(mob/living/M as mob, mob/user as mob)
 	if (!istype(M) || !istype(user))
 		return 0
+	if (istype(M,/mob/living/silicon/robot) && istype(user))
```

Comment:
something like `if(M == user)` would be better suited here
currently you are verifying user is a mob (which is done in the if in line 16 as well AND checking of your target is a robot/borg